src/ListSource.php

```diff
@@ -168,7 +168,14 @@ public function getQuery(array $options = []): QueryInterface {
 
     // Handle multilingual language fallback.
     if ($resolved_options['language'] && in_array('language_with_fallback', array_keys($index_fields))) {
-      $query->addCondition('language_with_fallback', $resolved_options['language']);
+      // Several languages.
+      if (is_array($resolved_options['language'])) {
```

Comment:
Why do we need to handle the case of a single or multiple languages? Let's just have an array of a single language. So always array.